src/components/tiptap/iframe/EmbedComponent.tsx

```diff
@@ -1,35 +1,41 @@
-import React, { useCallback } from 'react'
+import React, { useCallback, useState } from 'react'
 import { NodeViewWrapper } from '@tiptap/react'
 import { Resize } from '../image/resizeIcon'
 import { useAppState } from '@/hooks/useAppState'
 import { usePathname } from 'next/navigation'
+import { debounce } from '@mui/material'
 
 export const EmbedComponent = (props: any) => {
+  const [isResizing, setIsResizing] = useState(false)
+
   const handleMouseDown = useCallback(
     (event: React.MouseEvent<HTMLDivElement>) => {
       const parent = event.currentTarget.closest('.embed')
       const image = parent?.querySelector('div.iframe_container')
       if (!image) return
 
+      setIsResizing(true)
+
       const startSize = { x: image.clientWidth, y: image.clientHeight }
       const startPosition = { x: event.pageX, y: event.pageY }
 
-      const onMouseMove = (mouseMoveEvent: MouseEvent) => {
+      const onMouseMove = debounce((mouseMoveEvent: MouseEvent) => {
         props.updateAttributes({
           width: startSize.x - startPosition.x + mouseMoveEvent.pageX,
           height: startSize.y - startPosition.y + mouseMoveEvent.pageY,
         })
-      }
+      }, 1)
```

Comment:
I believe MUI uses `lodash.debounce`, which expects second argument in **milliseconds**.
So that would mean we are debouncing on each millisecond - adding it to a higher value like 10 milliseconds would be better for performance? Would that make the movement jittery?

Reply:
updated in the next commit.